src/disco/containers/InstallButton.js

```diff
@@ -143,9 +152,11 @@ export function mapDispatchToProps(dispatch) {
       });
     },
 
-    uninstall({ guid, installURL, slug }) {
+    uninstall({ guid, installURL, slug, name, type }) {
       const addonManager = new AddonManager(guid, installURL);
       dispatch({type: 'START_UNINSTALL', payload: {slug}});
+      const action = type === THEME_TYPE ? 'theme' : 'addon';
+      tracking.sendEvent({action, category: UNINSTALL_CATEGORY, label: name});
```

Comment:
It might be nice if the tracking library had `trackInstall`/`trackUninstall` and we could keep some of this in there.

```js
tracking.trackInstall({type, name});

// In tracking library
function addonTypeToAction(type) {
  return {
    ADDON_TYPE: 'addon',
    THEME_TYPE: 'theme',
  }[type] || 'invalid';
}

class Tracking {
  trackInstall({ type, name }) {
    this.sendEvent({action: addonTypeToAction(type), category: INSTALL_CATEGORY, label: name});
  }
}

```